amnestywebsite/humanity-theme
# Enable the core/navigation block and style

Ref: https://github.com/amnestywebsite/humanity-theme/issues/316

Re-enables the core/navigation block, adds a block style for a grey background and styles the block to match the old custom Menu block.

Flickity functionality for the front end is going to be handled at a later date in a separate ticket

**Steps to test**:
1. Edit a post
2. Open the block inserter and search for navigation (you may need to disable the branding plugin if it doesn't show)
3. Insert the block, and populate, it should match the styles of the old custom Menu block
4. Check and change the block style to grey background
5. Save and view on the front end

**Considerations**:
- RTL: Needs to be tested for RTL users to ensure the block displays correctly

**Example:**
http://bigbite.im/i/cnKyZT
